packages/promotion/src/models/campaign.ts

```diff
@@ -15,17 +16,16 @@ import {
 import CampaignBudget from "./campaign-budget"
 import Promotion from "./promotion"
 
+type OptionalRelations = "budget"
 type OptionalFields =
   | "description"
   | "currency"
   | "starts_at"
   | "ends_at"
-  | "deleted_at"
-  | DAL.EntityDateColumns
-
-type OptionalRelations = "budget"
+  | DAL.SoftDeletableEntityDateColumns
 
-@Entity()
+@Entity({ tableName: "campaign" })
+@Filter(DALUtils.mikroOrmSoftDeletableFilterOptions)
```

Comment:
todo: all models, align to the conventions `Something | null = null` except for the relations where there is no initializer